Core/src/ExponentialBackoff.php

```diff
@@ -56,18 +56,22 @@ class ExponentialBackoff
 
     /**
      * @param int $retries [optional] Number of retries for a failed request.
-     * @param callable $retryFunction [optional] returns bool for whether or not to retry
-     * @param callable $onRetryExceptionFunction [optional] runs before the $retryFunction. Unlike the $retryFunction,
-     *        this function isn't responsible to decide if a retry should happen or not,
-     *        but it gives the users flexibility to consume exception messages and add custom logic.
-     *        Function definition should match: function (\Exception $e, int $attempt, array &$arguments) : void
-     *        Ex: One might want to change headers on every retry, this function can be used to achieve
-     *        such a functionality.
-     * @param callable $onExecutionStartFunction [optional] runs before execution of the
-     *        execute function. Taken in $arguments as reference and thus gives users,
-     *        the flexibility to add custom logic before the execution of request
-     *        and override request / options in the $arguments.
-     *        Function definition should match: function (\Exception $e, int $attempt, array &$arguments) : void
+     * @param callable $retryFunction [optional] returns bool for whether or not
+     *  to retry
```

Comment:
Was this done by codesniffer? This is weird. The indentation before was correct 🤔

Reply:
It was my mistake, fixed in [9e155ff](https://github.com/googleapis/google-cloud-php/pull/5953/commits/9e155ff90545e3b7a65efa01492261db4cdc874e)